package.json

```diff
@@ -4,41 +4,53 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "start": "cross-env NODE_ENV=staging node src/index.js",
-    "prod": "cross-env NODE_ENV=production node src/index.js",
-    "dev": "cross-env NODE_ENV=development nodemon src/index.js",
-    "test": "cross-env NODE_ENV=staging && nyc mocha",
-    "lint": "eslint src"
```

Comment:
I have a point here, 

If you check on `develop` branch you will see that we had these scripts modified to support environment configuration chore, things like `cross-env NODE_ENV=production`.

Instead of totally removing them you can just modify them a little bit to work with this chore about ci-cd. And I think it's not that hard. 

Let it be like this.

```json
    "start": "cross-env NODE_ENV=staging node src/index.js",
    "prod": "cross-env NODE_ENV=production node src/index.js",
    "dev": "cross-env NODE_ENV=development nodemon src/index.js",
    "test": "cross-env NODE_ENV=staging & nyc mocha --recursive ./src/test",
    "lint": "eslint ./src --ext .js --fix"
```

Reply:
Fixed bro!!!!